st2api/st2api/controllers/v1/service_registry.py

```diff
@@ -0,0 +1,77 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import six
+
+from tooz.coordination import GroupNotCreated
+
+from st2common.services import coordination
+from st2common.exceptions.db import StackStormDBObjectNotFoundError
+from st2common.rbac import utils as rbac_utils
+
+__all__ = [
+    'ServiceRegistryGroupsController',
+    'ServiceRegistryGroupMembersController',
+]
+
+
+class ServiceRegistryGroupsController(object):
+    def get_all(self, requester_user):
+        rbac_utils.assert_user_is_admin(user_db=requester_user)
+
+        coordinator = coordination.get_coordinator()
+
+        group_ids = list(coordinator.get_groups().get())
+        group_ids = [group_id_.decode('utf-8') for group_id_ in group_ids]
```

Comment:
Is the underscore at the end of `group_id_` mean something?

Reply:
Yes, ``group_id`` variable is already defined in the method signature and I don't want to re-define / overwrite it in a list comprehension.

Reply:
Practice wise, we don't usually put an underscore at the end of a variable.  This is confusing and error prone. Can you rename `group_id_` to something else?  Maybe `item` just for this list comprehension.